examples/tests/backend_driver.py

```diff
@@ -153,47 +147,41 @@ def drive(backend, directories, python=[sys.executable], switches=[]):
 
 def parse_options():
     doc = (__doc__ and __doc__.split('\n\n')) or "  "
-    op = OptionParser(description=doc[0].strip(),
-                      usage='%prog [options] [--] [backends and switches]',
-                      epilog='\n'.join(doc[1:]))
-    op.disable_interspersed_args()
-    op.set_defaults(dirs='',
-                    clean=False, coverage=False, valgrind=False)
-    op.add_option('-d', '--dirs', '--directories', type='string',
-                  dest='dirs', help=dedent('''
-      Run only the tests in these directories; comma-separated list'''))
-    op.add_option('-b', '--backends', type='string', dest='backends',
-                  help=dedent('''
-      Run tests only for these backends; comma-separated list of
-      one or more of: agg, ps, svg, pdf, template, cairo,
-      Default is everything except cairo.'''))
-    op.add_option('--clean', action='store_true', dest='clean',
-                  help='Remove result directories, run no tests')
-    op.add_option('-c', '--coverage', action='store_true', dest='coverage',
-                  help='Run in coverage.py')
-    op.add_option('-v', '--valgrind', action='store_true', dest='valgrind',
-                  help='Run in valgrind')
-
-    options, args = op.parse_args()
-    switches = [x for x in args if x.startswith('--')]
-    backends = [x.lower() for x in args if not x.startswith('--')]
-    if options.backends:
-        backends += [be.lower() for be in options.backends.split(',')]
-
-    result = Bunch(
-        dirs=options.dirs.split(','),
-        backends=backends or ['agg', 'ps', 'svg', 'pdf', 'template'],
-        clean=options.clean,
-        coverage=options.coverage,
-        valgrind=options.valgrind,
-        switches=switches)
-    return (result)
+    parser = ArgumentParser(description=doc[0].strip(),
+                            epilog='\n'.join(doc[1:]))
+    parser.set_defaults(dirs='', clean=False, coverage=False, valgrind=False)
```

Comment:
The default for anything with `store_true` is already `False`. I'd just pass `default` to the two that need it.